File: Kritika Tripathi/Basics Of Programming/Getting Started/gcdlcm.cpp
#include <iostream>
using namespace std;

int main(){
    int num1, num2;
    cin >> num1 >> num2;

    //write your code here
    int gcd;
    int n = min(num1, num2);
    for(int i=1; i<=n; i++){
        if(num1%i==0 && num2%i==0){
            gcd=i;
        }
    }
    int lcm=(num1*num2)/gcd;

    cout<<"GCD: "<<gcd<<endl;
    cout<<"LCM: "<<lcm<<endl;
    return 0;
}

File: V.Vennela/Basics of Programming/gcd_lcm.java
import java.util.*;
    
    public class gcd_lcm{
    
    public static void main(String[] args) {
      // write your code here 
      Scanner sc = new Scanner(System.in);
      int a = sc.nextInt();
      int b = sc.nextInt();
      int x=0;
      if(a<b)
        x=a;
      else
        x=b;
      int i=x;
      while(i>0)
      {
        if(a%i==0 && b%i==0)
            break;
        i--;
      }
      System.out.println(i);
      System.out.println((a*b)/i);
     }
    }
